src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Translation\PlatformAdapter\PhraseApp\Bridge\Symfony\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $root = $treeBuilder->root('translation_adapter_phrase_app');
+
+        $root->children()
+            ->scalarNode('httplug_client')->defaultValue('httplug.client')->cannotBeEmpty()->end()
+            ->scalarNode('httplug_message_factory')->defaultValue('httplug.message_factory')->cannotBeEmpty()->end()
+            ->scalarNode('httplug_uri_factory')->defaultValue('httplug.uri_factory')->cannotBeEmpty()->end()
```

Comment:
The PhraseApp client is hard coupled to Guzzle :(

There is no need for httplug here.